client/my-sites/site-settings/test/utils.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Internal dependencies
+ */
+import {
+	wpcomFreePlan,
+	wpcomPremiumPlan,
+	wpcomBusinessPlan,
+	wpcomEcommercePlan,
+	wpcomEnterprisePlan,
+	wpcomVipPlan,
+	jpFreePlan,
+	jpPersonalPlan,
+	jpPremiumPlan,
+	jpBusinessPlan,
+	jpSecurityDailyPlan,
+	jpSecurityRealtimePlan,
+	jpCompletePlan,
+} from '../fixture';
+import { hasSiteAnalyticsFeature } from '../utils';
+
+describe( 'hasSiteAnalyticsFeature', () => {
+	it( 'returns undefined if the site is not defined', () => {
+		expect( hasSiteAnalyticsFeature() ).toBeUndefined();
+	} );
+
+	it( 'returns undefined if the site has no plan', () => {
```

Comment:
Just a question: Do you think we should add a test case for when `{ plan: undefined }`?

Reply:
It'd be similar than for `{}`, but it could be worth for `{ plan: null }`. Thanks! I'll add that in the next PR.